css/main.scss

```diff
@@ -27,10 +27,11 @@
  */
 
 @import "components--footer";
+@import "components--quote";
 
 /**
- * Layouts (sorted alphabetically)
+ * Layouts (order matters)
```

Comment:
In global layouts, I made a site-section class which i put on every section to be consistent with the margin beetwen sections, and I wanted to override this on some sections. This got refactored and order doesn't matter now tho